### PR TITLE
add Triphouse-project

### PR DESCRIPTION
**Pull request by MashaLoban**
add Triphouse-project, but I don't know how to align the text under the pictures

### CODE REVIEW

File: Triphouse-project/style.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+@media (max-width: 360px) {
+    .wrapper .wrapper__items {
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.wrapper__item img {
+    display: block;
+    margin: 0px auto;
```
Comment (ghost) [inline Triphouse-project/style.css]: Add height, for example 100px and some margin-bottom. 
Add will be prettier.
2021-04-08_22h25_36

File: Triphouse-project/style.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.wrapper {
+    margin: 0px auto;
+    text-align: center;
+    max-width: 1440px;
+
```
Comment (ghost) [inline Triphouse-project/style.css]: Make your code more clear.  Delete empty lines.

File: Triphouse-project/style.css (hunk 1 above)
Comment (MashaLoban) [reply Triphouse-project/style.css]: fixed

File: Triphouse-project/style.css (hunk 2 above)
Comment (MashaLoban) [reply Triphouse-project/style.css]: done

**Pull request merged**